src/components/signIn.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Button from 'react-bootstrap/Button'
+
+export default class Login extends Component {
+  render () {
+    return (
+      <form>
+        <h3>Sign In</h3>
+
+        <div className='form-group'>
+          <label>Username</label>
+          <input type='username' className='form-control' placeholder='Enter username' />
+        </div>
+
+        <div className='form-group'>
+          <label>Password</label>
+          <input type='password' className='form-control' placeholder='Enter password' />
+        </div>
+
+        <div className='form-group'>
+          <div className='custom-control custom-checkbox'>
+            <input type='checkbox' className='custom-control-input' id='customCheck1' />
+            <label className='custom-control-label' htmlFor='customCheck1'>Remember me</label>
+          </div>
+        </div>
```

Comment:
These could be separated into components. Boostrap has a React version. Or you could create the components yourself.